app/app.R

```diff
@@ -20,23 +20,54 @@ ui <- navbarPage(
     windowTitle = apptitle,
     theme = "styling.css",
     position="fixed-top", collapsible = TRUE,
-    tabPanel("Results", sidebarLayout(position = "right",
+    tabPanel("Results", sidebarLayout(position = "left",
       sidebarPanel(
-        numericInput("R",
-          "R:",
-          min = 0.5,
-          max = 10,
-          value = 2
+        dateInput("admissionDate",
```

Comment:
Cosmetic, but trying to use a single naming convention to facilitate review by several people. Could you use `snake_case` rather than `camelCase` throughout, esp in the names of the inputs?

So here `admissionDate` would become `admission_date` for instance.